src/api/java/atomicstryker/dynamiclights/client/IDynamicLightSource.java

```diff
@@ -0,0 +1,8 @@
+package atomicstryker.dynamiclights.client;
+
+import net.minecraft.entity.Entity;
+
+public interface IDynamicLightSource {
```

Comment:
This class **will not** be included in the product jar and was not included in the patched jar in the test. It is just served as `provided` dependency.
![image](https://cloud.githubusercontent.com/assets/7806504/19953190/8f54badc-a128-11e6-88c0-ac81b91938aa.png)